tests/run-tests.sh

```diff
@@ -39,4 +39,13 @@ fi
 
 
 
+
+jsight convert openapi json test01.jst > /tmp/test01.json
+if cmp --silent /tmp/test01.json ./expected/test01.json; then
```

Comment:
Let’s name the file **expected/openapi01.json**

I would also rename other files **expected/test0*** to **expected/jdoc0***

Reply:
todo